src/devices/zemismart.ts

```diff
@@ -368,6 +368,43 @@ const definitions: Definition[] = [
         description: 'Smart 20A outlet',
         extend: [identify(), tuya.modernExtend.tuyaOnOff({indicatorMode: true, onOffCountdown: true, childLock: true})],
     },
+    {
+        fingerprint: tuya.fingerprint('TS004F', ['_TZ3000_11pg3ima']),
+        model: 'ZMR4',
+        vendor: 'Zemismart',
+        description: 'Wireless switch with 4 buttons',
+        exposes: [
+            e.battery(),
+            e.action([
+                '1_single',
+                '1_double',
+                '1_hold',
+                '2_single',
+                '2_double',
+                '2_hold',
+                '3_single',
+                '3_double',
+                '3_hold',
+                '4_single',
+                '4_double',
+                '4_hold',
+            ]),
+        ],
+        fromZigbee: [fz.battery, tuya.fz.on_off_action],
+        toZigbee: [],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await endpoint.read('genBasic', [0x0004, 0x000, 0x0001, 0x0005, 0x0007, 0xfffe]);
```

Comment:
```suggestion
            await tuya.configureMagicPacket(device, coordinatorEndpoint);
```

Isn't this the same?